main.tex

```diff
@@ -1258,18 +1258,35 @@ \section{Conclusion}
 develop, and contribute to those tools within a conscientious and welcoming
 community.
 
-\acknowledgments
-
-Who to thank?  numfocus, GSOC, Python Software Foundation
-
-We thank the many web services that make developing and maintaining this package
-feasible, such as GitHub, Travis CI, Appveyor, CircleCI, and Read the Docs.
+\acknowledgments The \astropy community is supported by and makes use
+of a number of organizations and services outside the traditional
+academic community. We thank Google for financing and organizing the
+Google Summer of Code (GSoC) program, that has funded severals
+students per year to work on \astropy related projects over the
+summer. These students often turn into long-term contributors. We also
+thank NumFOCOS and the Python Software Foundation for financial
+support. Within the academic community, we thank
+institutions that make it possible that developers on
```

Comment:
sorry to be nitpicking on this, but I think it was better in the previous round to also mention astronomers. This current wording can falsely suggest that it's only people in the engineering departments who fall in this category, while the reality is that some institutions are indeed OK with their staff using their **science** time to write community software.